src/AWSS3.jl

```diff
@@ -698,7 +713,7 @@ See also [`AWSS3.s3_nuke_bucket`](@ref).
 function s3_delete_bucket(aws::AbstractAWSConfig, bucket; kwargs...)
     return parse(S3.delete_bucket(bucket; aws_config=aws, kwargs...))
 end
-s3_delete_bucket(a; b...) = s3_delete_bucket(global_aws_config(), a; b...)
+s3_delete_bucket(a; kwargs...) = s3_delete_bucket(global_aws_config(), a; kwargs...)
```

Comment:
May as well name the argument:
```suggestion
function s3_delete_bucket(bucket; kwargs...)
    return s3_delete_bucket(global_aws_config(), bucket; kwargs...)
end
```